house_x: 7
house_y: 7